### PR TITLE
Added post processing script

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#241

### Test plan
1. Download project directory 
2. Run script locally
3. Verify that the new files are conforming to spec in the ticket

Note: Tested against test projects from staging Terra staging area (specified in the ticket)

### CODE REVIEW

**Comment by aaclan-ebi** [comment]
couldn't add @amnonkhen yet as reviewer , @ESapenaVentura could you check the permissions 🤔

**Comment by ESapenaVentura** [comment]
It took just some time but he's got the permissions now :)

**Comment by ESapenaVentura** [comment]
It's still not modifying the filenames for the analysis_files

It's modifying the metadata in the file_descriptor but the filenames remain the same

New Checksums need to be generated

**Comment by ESapenaVentura** [comment]
There actually was no error!

LGTM

**Review by ESapenaVentura — APPROVED** [review]

File: src/post_processor.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+                    old_input_uuid = input['input_id']
+                    new_input_uuid = self.find_new_uuid(old_input_uuid)
+                    input['input_id'] = new_input_uuid
+
+                for output in link.get('outputs'):
+                    old_output_uuid = output['output_id']
+                    new_output_uuid = self.find_new_uuid(old_output_uuid)
+                    output['output_id'] = new_output_uuid
+
+                for protocol in link.get('protocols'):
```
Comment (ESapenaVentura) [inline src/post_processor.py:243]: Please test this, doing it rushed

**Comment by aaclan-ebi** [comment]
Please just comment directly on the code, if you have suggestions.

**Pull request merged**